README.md

```diff
@@ -15,7 +15,7 @@ gumpi requires at least OpenMPI 3.0 on the provider machine. Since the current L
 
 If the hub listens at `127.0.0.1:61622` and you want to spawn `12` processes, enter `gumpi` project root, execute:
 ```
-cargo run -- --hub 127.0.0.1:61622 --job job.toml -n 12
+cargo run -- --hub 127.0.0.1:61622 --job job.toml -n 12 -t 1
```

Comment:
It'd be good if you docuemented what `-t` implies here as well (and not only in the `Tutorial.md`).